website/src/app/kb/deploy/resources/readme.mdx

```diff
@@ -46,6 +46,26 @@ From there, you can select the type of Resource you want to create:
   Resource.
 </Alert>
 
+#### Routing order for overlapping addresses
+
+When multiple Resources' addresses overlap, the Resource with the more specific
+address will be used.
+
+For CIDR Resources, an address with a longer prefix is more specific than a
+shorter one. For example: `10.0.0.0/16` is more specific than `10.0.0.0/8`. IP
+Resources are essentially addresses with `/32` prefix and thus always more
+specific than any other CIDR. be used.
```

Comment:
```suggestion
specific than any other CIDR.
```

Couldn't make a multi-line comment when I made the original suggestion (was on my phone), so this one got lost.